models/projection/powder/mantid_projection_calculator.py

```diff
@@ -1,22 +1,15 @@
-from math import sqrt,cos
 
-from mantid.simpleapi import SofQWNormalisedPolygon,mtd
+from mantid.simpleapi import AnalysisDataService,ConvertToMD
 
 from models.projection.powder.projection_calculator import ProjectionCalculator
+from models.workspacemanager.mantid_workspace_provider import MantidWorkspaceProvider
 
 
 class MantidProjectionCalculator(ProjectionCalculator):
-    def calculate_projection(self, input_workspace, output_workspace, qbinning, axis1, axis2):
-        if axis1 == 'Energy' and axis2 == '|Q|':
-            #TODO is EMode always direct ?
-            SofQWNormalisedPolygon(InputWorkspace=input_workspace, OutputWorkspace=output_workspace,
-                                                                QAxisBinning=qbinning,EMode='Direct')
-
-    def calculate_suggested_binning(self, input_workspace):
-        return '0,.1,10'
-        #TODO AskInstrument scientist about calculating scattering angles
-        input_workspace = mtd[input_workspace]
-        ei = float(input_workspace.getRun().getLogData('Ei').value)
-        ki = sqrt(ei/2.07)
-        kf = sqrt((ei-0.0)/2.07)
-        Qx=ki*1-cos(minTheta)*kf
\ No newline at end of file
+    def calculate_projection(self, input_workspace, output_workspace, axis1, axis2):
+        import random,string
```

Comment:
Is there a reason this has been modified when the PR is related to implementing zooming? This seems completely unrelated.